packages/dev/loaders/src/BVH/bvhLoader.ts

```diff
@@ -0,0 +1,388 @@
+import type { IAnimationKey } from "core/Animations";
+import { Animation } from "core/Animations/animation";
+import { Bone } from "core/Bones/bone";
+import { Skeleton } from "core/Bones/skeleton";
+import { Axis } from "core/Maths/math.axis";
+import { Matrix, Quaternion, Vector3 } from "core/Maths/math.vector";
+import type { Scene } from "core/scene";
+import type { Nullable } from "core/types";
+import type { BVHLoadingOptions } from "./bvhLoadingOptions";
+
+/**
+ * Original code from: https://github.com/herzig/BVHImporter/
+ *
+ * Ivo Herzig, 2016
+ * MIT License
+ */
+export class BVHParser {
```

Comment:
This class holds only static members. The class itself is not needed - all needed functions can be imported from the module context.
We try to avoid this empty-static only-class with new implementation, though I know we have this pattern in different areas in the framework.

Reply:
@RaananW, I made some changes, is that what you had in mind?